docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -1,7 +1,68 @@
 [[fix-common-cluster-issues]]
 == Fix common cluster issues
 
-This guide describes how to fix common problems with {es} clusters.
+This guide describes how to fix common errors and problems with {es} clusters.
+
+[discrete]
+=== Error: disk usage exceeded flood-stage watermark, index has read-only-allow-delete block
+
+This error indicates a data node is critically low on disk space and has reached
+the <<cluster-routing-flood-stage,flood-stage disk usage watermark>>. When a
+node reaches this watermark, {es} blocks writes to any index with a shard on the
+node to prevent a full disk. If the block affects related system indices, {kib}
+and other {stack} features may become unavailable.
+
+You can use the <<cat-allocation,cat allocation API>> to get the current
+available disk space (`disk.avail`) and disk usage percentage (`disk.percent`)
+for each node.
+
+[source,console]
+----
+GET _cat/allocation?v=true
+----
+
+{es} will automatically remove the write block when the node goes below the
```

Comment:
Could we also mention that Elasticsearch will be working to move shards away from the overfull node until it drops below the high watermark, assuming there's somewhere else to put them? Ideally users should check that shards really are moving onto the other nodes, and if they're not then they should use the cluster allocation explain API to understand why not.